### ISSUE TITLE
importing vue-password is ~780k!

### ISSUE THREAD

**Issue by zhegwood** [skegel13/vue-password#21]
I started using this tonight and realized after importing vue-password my js webpack bundle went from 450k to 1.23M.  Need to do something about the package size, my friend.

**Comment by skegel13** [comment]
The library that is used to score the passwords is very large. This is mentioned in the README. If you want to use a smaller library (or server-side) library to score the passwords, you can.

https://github.com/skegel13/vue-password#custom-password-strength-library

**Comment by rcbrandaops** [comment]
I'm using VuePassword on my app and it is importing zxcvbn. I'm not registering it globally. I'm only importing the ```VuePassword``` component and using my own algorithm to score the passwords. 

To make a test, I removed both VuePassword and zxcvbn through ```npm uninstall```. Than, I deleted the ```node_modules``` folder and ```package-lock.json``` and finally installed VuePassword again.

Somehow, the library was imported and it creates a requires in ```package-lock.json```:

```
"requires": {
        "zxcvbn": "^4.4.2"
}
```
I don't know if I'm doing something wrong, but that's the case.
